src/components/ResultsBody/PaginatedList/VenueCard/venueCard.module.css

```diff
@@ -0,0 +1,75 @@
+.wrapper {
+  min-width: 282px;
+  min-height: 228px;
+  box-shadow: 0 0 0.4vw var(--secondary-light);
+}
+
+:root {
+  --background-url: url("default-image.jpg");
+}
+
+.pictureFrame {
+  height: 80%;
+  width: 100%;
+  background-size: cover;
+  background-position: center;
+  position: relative;
+  background-image: var(--background-url);
+}
+
+.infoBar {
+  background-color: var(--background-gray);
+  min-height: 28px;
+  width: 100%;
+  height: 20%;
+}
+
+.heartBox {
+  width: 2.22vw;
+  height: 2.22vw;
+  background-color: black;
+  opacity: 0.63;
+  position: absolute;
+  top: 0;
+  border-radius: 0 0 10px 0;
+}
+
+.nameBox {
+  height: 2.22vw;
+  width: fit-content;
+  background-color: black;
+  opacity: 0.63;
+  position: absolute;
```

Comment:
position absolute is one of the worst things to do. It is a bad practice. Don't use it unless there is no other option, but such a case is very rare.

Reply:
done